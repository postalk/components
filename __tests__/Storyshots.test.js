import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  storyNameRegex: /^((?!.*?DontTest).)*$/,
  storyKindRegex: /^((?!.*?DontTest).)*$/,
  configPath: '.storybook/config.test.js'
})
