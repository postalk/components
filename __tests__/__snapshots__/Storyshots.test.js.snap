// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Card Default 1`] = `
<div>
  <div
    class="cardRoot"
    id="key1"
    style="left: 68px; top: 212px; height: -38px;"
  >
    <div
      class="card"
    >
      <ul
        class="list"
      >
        <li
          class="item"
        >
          
    とんこつラーメン
  
        </li>
        <li
          class="item"
        >
          
    油そば野菜油増し増しスペシャル
  
        </li>
        <li
          class="item"
        >
          
    うどん
  
        </li>
      </ul>
       
      <!---->
    </div>
     
    <!---->
     
    <!---->
  </div>
</div>
`;

exports[`Storyshots Cards Default 1`] = `
<div
  class="canvas"
>
  <div
    class="cardRoot"
    id="key1"
    style="left: 68px; top: 212px; height: -38px;"
  >
    <div
      class="card"
    >
      <ul
        class="list"
      >
        <li
          class="item"
        >
          
    とんこつラーメン
  
        </li>
        <li
          class="item"
        >
          
    油そば野菜油増し増しスペシャル
  
        </li>
        <li
          class="item"
        >
          
    うどん
  
        </li>
      </ul>
       
      <!---->
    </div>
     
    <!---->
     
    <!---->
  </div>
  <div
    class="cardRoot"
    id="key2"
    style="left: 452px; top: 140px; height: -38px;"
  >
    <div
      class="card"
      style="background-color: rgb(255, 166, 166); border-color: #f39393;"
    >
      <h2
        class="headline"
      >
        適当に遊んで暮らす毎日
      </h2>
       
      <!---->
    </div>
     
    <!---->
     
    <!---->
  </div>
  <div
    class="cardRoot isDisabled"
    id="key3"
    style="left: 68px; top: 140px; height: -38px;"
  >
    <div
      class="card"
      style="background-color: rgb(156, 232, 232); border-color: #8ee8e8;"
    >
      <h2
        class="headline"
      >
        福岡でなに食べるか
      </h2>
       
      <!---->
    </div>
     
    <!---->
     
    <div
      class="disabled-text"
    >
      
    selected by someone
  
    </div>
  </div>
  <div
    class="cardRoot isDisabled"
    id="key4"
    style="left: 452px; top: 212px; height: -38px;"
  >
    <div
      class="card"
    >
      <div
        class="text"
      >
        ようやくの思いで笹原を這い出すと向うに大きな池がある。吾輩は池の前に坐ってどうしたらよかろうと考えて見た。別にこれという 分別 （ ふんべつ ） も出ない。
      </div>
       
      <!---->
    </div>
     
    <!---->
     
    <div
      class="disabled-text"
    >
      
    selected by someone
  
    </div>
  </div>
  <div
    class="cardRoot"
    id="key5"
    style="left: 836px; top: 140px; height: -38px;"
  >
    <div
      class="card"
      style="background-color: rgb(253, 249, 155); border-color: #f9f116;"
    >
      <h2
        class="headline"
      >
        My Profile
      </h2>
       
      <!---->
    </div>
     
    <!---->
     
    <!---->
  </div>
  <div
    class="cardRoot isImage"
    id="key6"
    style="left: 836px; top: 212px; height: -38px;"
  >
    <div
      class="card"
      style="width: -42px; height: -42px;"
    >
      <img
        class="image"
        src="https://i.gyazo.com/4215f0df9b4c256b39afd12327f40277.png"
      />
       
      <!---->
    </div>
     
    <!---->
     
    <!---->
  </div>
   
  <!---->
   
  <!---->
   
  <!---->
   
  <div />
</div>
`;

exports[`Storyshots Centered Default 1`] = `
<div
  class="wrapper"
>
  <div
    class="centered"
  >
    I’m centered
  </div>
</div>
`;

exports[`Storyshots Input Default 1`] = `
<div
  class="wrapper"
>
  <div
    class="centered"
  >
    <div
      class="wrapper"
    >
      <div
        class="centered"
      >
        <div
          style="width: 300px;"
        >
          <textarea
            class="input"
            placeholder="Type something here..."
            style="resize: none; overflow: hidden; height: 30px;"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Shortcuts Default 1`] = `<div />`;

exports[`Storyshots Shortcuts Selected 1`] = `<div />`;
